src/metrics-modeling/metricbot/__main__.py

```
@@ -0,0 +1,82 @@
+            return run_week(
+                sport=args.sport,
+                dry_run=args.dry_run,
+                dump_intermediate=args.dump_intermediate,
+                season_year=args.season_year,
+                week=args.week,
+                prior_tail=args.prior_season_tail,
+                publish=args.publish,
+                legacy_extraction=args.legacy_extraction,
+            )
+        except Exception as ex:  # noqa: BLE001 — CLI boundary: fail loudly, exit nonzero
+            logging.getLogger("metricbot").error("Pipeline failed: %s", ex)
```

Ruff (0.16.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)